composer.json

```diff
@@ -20,7 +20,7 @@
         "duncan3dc/object-intruder": "^0.3.0",
         "duncan3dc/php-ini": "^0.1.0",
         "mockery/mockery": "^1.0",
-        "phpunit/phpunit": "^5.7.10",
+        "phpunit/phpunit": "^5.7.10|^6.5|^7.0",
```

Comment:
I don't want to have to support this many versions of PHPUnit at once, I normally just keep it to to one major (version 5 at the moment)

Reply:
According to the [PHPUnit website](https://phpunit.de/), the supported PHP versions are mapped for different PHPUnit version.
Maybe we can keep the two versions are ```^5.7.10|^6.5``` and it can support the ``php-5.6+```.

Reply:
But I'll have to support different bugs/features/issues/etc caused by having multiple versions in use. I don't want issues opening where the library isn't working correctly with PHPUnit 5.7, and the other issues where it's not working correctly with PHPUnit 6.0, etc

Reply:
Ok. I agree that.

Reply:
I keep the version being the ```^5.7.10```.